core/controllers/base.py

```diff
@@ -291,6 +312,15 @@ def render_json(self, values):
     def render_template(
             self, filename, iframe_restriction='DENY',
             redirect_url_on_logout=None):
+        """Prepares an HTML response to be sent to the client.
+
+        Args:
+            filename: str. The template filename, related to the
+                           templates directory.
```

Comment:
This should be indented by 4 from the previous line (see other files). Ditto below.